packages/beacon-node/src/network/libp2p/index.ts

```diff
@@ -1,7 +1,8 @@
 import {PeerId} from "@libp2p/interface";
 import {Registry} from "prom-client";
 import {ENR} from "@chainsafe/enr";
-import {identify} from "@libp2p/identify";
+// TODO: We should use this fork until https://github.com/libp2p/js-libp2p/pull/2387
```

Comment:
- Related to https://github.com/libp2p/js-libp2p/issues/2389

We needed a way to disable identify-push, since on fork boundaries we run into this issue.

This fork just deletes the identify-push code.
- Temporary fix until https://github.com/libp2p/js-libp2p/pull/2387 is merged